src/components/profile/style/profile.less

```diff
@@ -104,4 +96,216 @@
       left: 65px;
     }
   }
+   
+   .profile-icons {
+    width: auto;
+    display: inline-block;
+    overflow-x: auto;
+    left: 60px;
+    position: absolute;
+   
+
+    @media only screen and (max-width: 780px){
+      left: 0px; 
+      width: 100%;
+      height: 30px;
+     }
+   }
+
+   .wrapper {
+    position: relative;
+    display: inline-block;
+    width: 700px;
+}
+
+   .wrapper div {
+    float: left;
+}
+
+  .ga-profile-elevation-difference {
+   color: black; 
+   
+
+    .ga-profile-icon-text { 
+      position: relative;
+      font-size: 0.9em;
+      width: 50px;
+      padding-left: 1px;
+     }
+
+     .fa {
+    
+     
+     }
+    
+     .fa-resize-vertical::before {
+      position: relative;
+      font-size: 0.9em;  
+     }
+}
+  
+  .ga-profile-elevation-up {
+   color: black; 
+  
+   
+   .ga-profile-icon-text {
+    position: relative;
+    font-size: 0.9em;
+    padding-left: 4px;
+    width: 50px;
+   }
+   
+    .fa {
+    }
+
+    .fa-ga-elevation-up::before {
+     position: relative;
+     font-family: FontAwesome;
```

Comment:
it seems there is a lot of duplicate css rules in this file can yout try to factorize a bit please.